### PR TITLE
Added app basic router and snapshot test

### PR DESCRIPTION
**Pull request by AntonioDaria**
Added router as per trello card :
https://trello.com/c/4NSnFyfr/37-react-router-implement-react-router-on-core-codebase

### CODE REVIEW

**Review by nicholasmorgan — COMMENTED** [review]
1. I think that that each page should be inside its own file in a "pages" folder rather than in the App.js file.

2. I think we should also be using arrow functions e.g.

```
  function Home() {
    return <h2>Home Page</h2>;
  }
```
to become:

```
  const Home = () => {
    return <h2>Home Page</h2>;
  }
```

**Comment by Sandvich** [comment]
Perhaps a templating thing like with Flask. It makes sense for the `app.js` file to define the routes, but it'll quickly become unmanageable if you try to have all of the content in there too.
For a python example of what I mean, see: https://github.com/Sandvich/puppykit/blob/master/main.py

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,50 @@
...
+            <li>
+              <Link to="/about">About</Link>
+            </li>
+          </ul>
+        </nav>
+
+        {/* A <Switch> looks through its children <Route>s and
+            renders the first one that matches the current URL. */}
+        <Switch>
+          <Route path="/about">
```
Comment (nicholasmorgan) [inline src/App.js:37]: can this be done as one line? e.g.  `<Route path="/about" component={About} />`

**Comment by AntonioDaria** [comment]
Refactored as per review suggestions

**Comment by nicholasmorgan** [comment]
Final picky thing. Can you name your files in PascalCase. e.g AboutPage.js

**Comment by AntonioDaria** [comment]
> Final picky thing. Can you name your files in PascalCase. e.g AboutPage.js

I can do but I was following the convention we used in the last Garage Project:
files in the pages folder (Camel case)
files in the components folder (Pascal case)
Not sure what is the best but I went for what I was used to. Open to discussion as long as we agree on one style :)

**Comment by nicholasmorgan** [comment]
Ok, cool yeah agreed

**Review by nicholasmorgan — APPROVED** [review]

**Review by jeremybeucler-ibm — APPROVED** [review]

**Review by Sandvich — APPROVED** [review]

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201005-102730 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201005-102730") to [dev](https://console.bluemix.net/apps/01f9afb2-7e50-4304-b09a-0c0625aeb700?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20201019-105852 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20201019-105852") to [test](https://console.bluemix.net/apps/d1eb8017-1150-4180-bc64-b3548b581508?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201030-114007 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201030-114007") to [dev](https://console.bluemix.net/apps/a97cb91f-c6d0-45b9-8ec5-7691f0a32356?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201030-121315 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201030-121315") to [dev](https://console.bluemix.net/apps/ce47e4c4-e0aa-42cb-8e05-85fc971d83ea?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201103-192620 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201103-192620") to [dev](https://console.bluemix.net/apps/5fb4cd50-9294-4bce-bfcb-7b2b580ec0b6?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201112-174509 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201112-174509") to [dev](https://console.bluemix.net/apps/d438097c-ddb4-45e8-bdf9-4442713e77d2?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201112-182845 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201112-182845") to [dev](https://console.bluemix.net/apps/6b85ab02-2dcf-4a2c-bc16-751d4e95a935?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201113-150421 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201113-150421") to [dev](https://console.bluemix.net/apps/ccf8c6a4-8244-4400-8c42-7677e882a220?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201124-122718 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201124-122718") to [dev](https://console.bluemix.net/apps/e7ad1c09-212f-4f68-9639-8073f4cf1781?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201202-101709 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201202-101709") to [dev](https://console.bluemix.net/apps/56ba9f85-8947-4840-bdf0-ccef4ecd4149?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201217-145052 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201217-145052") to [dev](https://console.bluemix.net/apps/33545de7-22de-43a2-be66-d6f6744fc5b9?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210205-152747 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210205-152747") to [test](https://console.bluemix.net/apps/1d38058e-0941-478d-ab73-1c5c09e8e146?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210303-110944 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210303-110944") to [prod](https://console.bluemix.net/apps/f1598887-083c-4cac-a3a3-4950b5d061d5?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:prod"), including commit 81059fa79debb725b3d6daf442aebd9197aec50c

**Pull request merged**